packages/cdk-routing-split/src/add/handler.ts

```diff
@@ -2,24 +2,43 @@ import { APIGatewayEvent, Context } from 'aws-lambda';
 import { createLogger, Logger, serializeError } from '@daysmart/aws-lambda-logger';
 import { action } from './action';
 import { HttpError } from '../http-error';
+import { AddRequest, Key } from './interface';
 
-export const handler = async (event: APIGatewayEvent, context: Context): Promise<any> => {
+export const add = async (event: APIGatewayEvent, context: Context): Promise<any> => {
     let logger!: Logger;
 
     try {
         logger = createLogger(process.env.DEBUG === 'true', context.awsRequestId);
         logger.debug('add event', { event });
 
+        const request: AddRequest = JSON.parse(event.body as string);
+        validateRequest(request);
+
+        await action(request);
+
         return {
             statusCode: 200,
-            body: action(),
+            body: 'Success',
```

Comment:
why is this returning a string 'Success' ? i  would expect at most a bool, though I think returning a bool may be a preference of mine (a functional programming technique) but i think we've just removed the body in cases like this in the past, because the 200 indicates the success on its own and anything else had seemed redundant.